### PR TITLE
libwebm 1.0.0.28

### PR DESCRIPTION
**Pull request by chenrui333**
[`action-homebrew-bump-formula`](https://github.com/dawidd6/action-homebrew-bump-formula)

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by carlocab** [comment]
```
Error: An exception occurred within a child process:
  Errno::ENOENT: No such file or directory - sample
```
Seems to come from here:

https://github.com/Homebrew/homebrew-core/blob/4c0e58002f28c7389140a23fcec5c82771212efa/Formula/libwebm.rb#L26

**Comment by cho-m** [comment]
https://github.com/webmproject/libwebm/commit/207cd80a8673b6e51c81d65207a3c2fdf2226116
- sample → **mkvparser_sample**
- sample_muxer → **mkvmuxer_sample**

File: Formula/libwebm.rb [hunk 1]
```diff
@@ -23,9 +23,12 @@ def install
...
       system "make"
       lib.install "libwebm.a"
-      bin.install %w[sample sample_muxer vttdemux webm2pes]
+      bin.install %w[mkvparser_sample mkvmuxer_sample vttdemux webm2pes]
     end
     include.install Dir.glob("mkv*.hpp")
+    (include/"mkvmuxer").install Dir.glob("mkvmuxer/mkv*.h")
+    (include/"common").install Dir.glob("common/*.h")
+    (include/"mkvparser").install Dir.glob("mkvparser/mkv*.h")
     include.install Dir.glob("vtt*.h")
```
Comment (carlocab) [inline Formula/libwebm.rb:32]: Can we try to see if there's a `make install` now?

Comment (chenrui333) [reply Formula/libwebm.rb:32]: There is no `make install`?

Comment (cho-m) [reply Formula/libwebm.rb:32]: Only for the new WebM Parser API https://github.com/webmproject/libwebm/commit/40da750807003000460e38f92fcb9d887834d715, which is disabled by default.
Installed files on local build with `-DENABLE_WEBM_PARSER:BOOL=ON`:
```
├── include
│   └── webm
│       ├── buffer_reader.h
│       ├── callback.h
│       ├── dom_types.h
│       ├── element.h
│       ├── file_reader.h
│       ├── id.h
│       ├── istream_reader.h
│       ├── reader.h
│       ├── status.h
│       └── webm_parser.h
└── lib
    └── libwebm.a
```

**Comment by carlocab** [comment]
Oops. Let's rebase this one then merge it.

**Review by carlocab — DISMISSED** [review]

**Comment by carlocab** [comment]
Tests were skipped, so adding `automerge-skip` here until we can run the tests.

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**